qlib/finco/test_sumarize.py

```diff
@@ -1,9 +1,20 @@
 import unittest
```

Comment:
This file should be placed in qlib/tests/finco/

Reply:
fixed